### PR TITLE
Changed div to button for octo-sidebar-item

### PR DESCRIPTION
**Pull request by anirudhprabhakaran3**
#### Summary

Converted octo-sidebar-item from div to button
#### Ticket Link

https://github.com/mattermost/focalboard/issues/1513

### CODE REVIEW

**Comment by hahmadia** [comment]
@jespino Friendly Bump

File: webapp/src/components/sidebar/sidebarBoardItem.tsx [hunk 1]
```diff
@@ -160,16 +160,16 @@ const SidebarBoardItem = React.memo((props: Props) => {
...
-            </div>
+            </button>
             {!collapsed && boardViews.length === 0 &&
                 <div className='octo-sidebar-item subitem no-views'>
                     <FormattedMessage
                         id='Sidebar.no-views-in-board'
                         defaultMessage='No pages inside'
                     />
-                </div>}
+                </button>}
```
Comment (jespino) [inline webapp/src/components/sidebar/sidebarBoardItem.tsx:170]: This is wrong, you are closing a button, but opening a div. I guess it was as part of reverting the last change.

**Review by jespino — CHANGES_REQUESTED** [review]
A tiny error, otherwise looks good to me.

**Comment by hahmadia** [comment]
Could we also get the broken tests fixed. Thanks

**Review by hahmadia — APPROVED** [review]

**Comment by hahmadia** [comment]
Can you please fix the broken cypress test? After that, you can re-request a review.

**Comment by emilyacook** [comment]
Thanks for participating in Hacktoberfest! You can claim your sticker set here: https://get.printfection.com/hacktober21/4144583267 @anirudhprabhakaran3

**Comment by anirudhprabhakaran3** [comment]
Hello! I'll finish working on the issue by tomorrow, and then claim the sticker set if that's okay. Thanks!

**Comment by anirudhprabhakaran3** [comment]
@hahmadia could you approve the workflows? I'll work on the tests if any fail

**Comment by hahmadia** [comment]
Broken Tests can be seen now

**Comment by anirudhprabhakaran3** [comment]
Hello!
So i was trying to fix the cypress tests. However, when i tried to run the tests on the main branch, I'm still getting the same error. Is the issue in the cypress test itself?

**Pull request closed without merging**